### ISSUE TITLE
Warn message when users fetch certain goods

### ISSUE THREAD

**Issue by screeper** [athena-logistics/athena-backend#410]
I am thinking specifically of c&m. When users pick up goods from c&m we could give them a warning message not to forget the label.

**Comment by maennchen** [comment]
@screeper You mean a warning message that is displayed in the Move form if an item is selected?

**Comment by screeper** [comment]
Really? But what about the c&m stuff in the cooling trailers Gallusplatz, Gallusstrasse etc.? And what about the sales point Laurenzen?

**Comment by screeper** [comment]
But than you dont know how many boxes are left in a cooling unit for which vendor point. So if for example Gallusplatz uses all boxes of beer cups and Webergasse still has  five left you just see 5 boxes in the logistics center... Or am i thinking wrong? 🤔😋

**Comment by maennchen** [comment]
@screeper You're not wrong. The whole solution is a trade-off to make things as simple as possible for vendors so that we have a chance that they use it correctly. There's some details that can't be fulfilled by choosing the easiest approach.
